app/views.py

```diff
@@ -267,6 +269,8 @@ def activity():
             return redirect(url_for('views.activity'))
 
     return render_template('activity.html', **{
+        'user': current_user,
+        'QUESTIONNAIRES': QUESTIONNAIRES,
```

Comment:
We already inject this into all our templates, so it doesn't need to be passed in explicitly.  See https://github.com/GovLab/noi2/blob/master/app/factory.py#L152 .